src/Components/NavBar/NavItem.tsx

```diff
@@ -64,13 +76,13 @@ export const NavItem: React.FC<NavItemProps> = ({
         </MenuContainer>
       )}
 
-      {showOverlay && <Overlay />}
+      {showOverlay && <Overlay hover={hover} />}
     </Box>
   )
 }
 
 const MenuContainer = styled(Box)`
   position: absolute;
-  transform: translate(-80%);
-  padding-top: 18px;
+  transform: translateX(-90%);
+  top: 63px;
```

Comment:
Positions the drop-down menus a little more precisely, so they get closer to the right edge of the nav item. Not entirely ideal, pushing things around like but can find a more ideal positioning mechanism in a refactor pass.